sympy/simplify/powsimp.py

```diff
@@ -163,9 +163,9 @@ def recurse(arg, **kwargs):
             # allow 2**x/4 -> 2**(x - 2); don't do this when b and e are
             # Numbers since autoevaluation will undo it, e.g.
             # 2**(1/3)/4 -> 2**(1/3 - 2) -> 2**(1/3)/4
-            if (b and b.is_Number and not all(ei.is_Number for ei in e) and \
-                    coeff is not S.One and
-                    b not in (S.One, S.NegativeOne)):
+            if (b and b.is_Number and b.is_rational and not all(
```

Comment:
If you need an actual rational number you should use is_Rational or isinstance(b, Rational). is_rational will also be True for things like `Symbol('x', rational=True)`.

Reply:
@asmeurer Fixed 😄